### PR TITLE
Update purchase tester

### PR DESCRIPTION
**Pull request by aboedo**
Updated the `PurchaseTester` app to the latest syntax. 
Also, it looks like we now require importing StoreKit manually, perhaps that's something that we can improve. 
Reminder that this app is for our internal testing only, so we should just make it work as we fix issues along the way.

### CODE REVIEW

File: IntegrationTests/PurchaseTester/Podfile.lock [hunk 1]
```diff
@@ -1,22 +1,16 @@
 PODS:
-  - Purchases (3.13.0-SNAPSHOT):
-    - PurchasesCoreSwift (= 3.13.0-SNAPSHOT)
-  - PurchasesCoreSwift (3.13.0-SNAPSHOT)
```
Comment (aboedo) [inline IntegrationTests/PurchaseTester/Podfile.lock:4]: bye bye `PurchasesCoreSwift`

File: IntegrationTests/PurchaseTester/PurchaseTester/SwiftPaywall.swift [hunk 2]
```diff
@@ -362,7 +363,7 @@ class SwiftPaywall: UIViewController {
             ])
         
         // The offerings loading indicator
-        offeringLoadingIndicator = UIActivityIndicatorView(style: .gray)
+        offeringLoadingIndicator = UIActivityIndicatorView(style: .medium)
```
Comment (aboedo) [inline IntegrationTests/PurchaseTester/PurchaseTester/SwiftPaywall.swift:366]: these are unrelated, but we've had these build warnings for a while and they're annoying

**Review by vegaro — APPROVED** [review]

File: IntegrationTests/PurchaseTester/Podfile.lock (hunk 1 above)
Comment (vegaro) [reply IntegrationTests/PurchaseTester/Podfile.lock:4]: it won't be missed

Comment (taquitos) [reply IntegrationTests/PurchaseTester/Podfile.lock:4]: Lol

**Pull request merged**